packages/core/src/Table/ExpandableTableRow.js

```diff
@@ -0,0 +1,63 @@
+import propTypes from '@dhis2/prop-types'
+import React, { useState } from 'react'
+import { ExpandToggleCell } from './ExpandableTableRow/ExpandToggleCell.js'
+import { TableRow } from './TableRow.js'
+
+/**
+ * @module
+ * @param {TableRow.PropTypes} props
+ * @returns {React.Component}
+ * @example import { TableRow } from '@dhis2/ui'
+ * @see Live demo: {@link /demo/?path=/story/table--expandable-table-rows|Storybook}
+ */
+export const ExpandableTableRow = ({
```

Comment:
@cooper-joe this component provides the default "Click to show more" functionality. If you want to open show expanded content programatically and/or need any other custom stuff, just build your own using the `TableRow`